### DISCUSSION TITLE
Building for Windows

### DISCUSSION THREAD

**Discussion by electron-shepherd** [eteran/nedit-ng#334] [Q&A] [answered]
I've used NEDIT for over a decade in the Linux environment for development using a hardware description language.  I'm extremely unfamiliar with building anything in a Windows environment, and am reaching out to see if someone might have any insight to help me make this work - as my new company only uses Windows, and I'd love to continue using NEDIT if possible.
I've installed QT (the latest, QT Creator 8.0.0, based on Qt 6.4.1, MSVC 2019, x86_64).  I've installed Visual Studio, and was able to find the MSVC, as indicated previously.  I have the latest boost libraries unzipped and the "BOOST_ROOT" variable pointed to them (those errors did appear initially, and then went away after setting that environment variable).  I went ahead and downloaded/installed the latest cmake, but it seems the default with QT is already beyond what is asked for in the requirements.  I've attempted to make changes in the QT kit configuration ... but I'm extremely unfamiliar with QT, and no matter what I do, I just get an error from QT that always states:
No CMake configuration found!
...with build errors associated with CMake not being found.
As a newbie to QT, and building anything on Windows ... any tips/tricks/pointers would be appreciated.
Thanks!
![image](https://user-images.githubusercontent.com/69281476/183665878-7e9f670b-36e2-4570-830d-059294acf045.png)

**Comment by eteran** [comment]
Hello! Love to see a fellow NEdit lover here :-).

I'll take a look at my instructions and see if they need a refresh, perhaps even a quick video of start to finish so there's no room for error.

**Reply by eteran** [reply]
Sorry for the delay, I haven't forgotten about this!

**Comment by eteran** [comment, marked as answer]
Wow, 4 years later :-/.

So looking at the errors, the issue WAS that you were attempting to compile with Qt6, but only Qt5 was supported at the time. Qt6 is now supported, so it should "just work"
